project/Dependencies.scala

```diff
@@ -518,4 +518,11 @@ object Dependencies {
 
   // Version of the swagger UI to write into config files
   val swaggerUiVersion = swaggerUiV
+
+  val perfDependencies = List(
+    "io.circe" %% "circe-java8" % circeV,
+    "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingV,
```

Comment:
This is OK, but I think it's time this were just added to `val commonDependencies` and the other copies of this line removed.

Reply:
I will try and do that 👍